### PR TITLE
Fix import-DB for the testnet edge case

### PR DESCRIPTION
**Pull request by iulianpascalau**
This PR fixes the edge case found in testnet when a shard stopped including metaheaders for 2 epochs

- made the storage resolvers open 2 epochs in advance. Also, the maximum active persisters value is set to the maximum value it can hold (config.StoragePruning.NumEpochsToKeep) in import-db mode

### CODE REVIEW

File: dataRetriever/storageResolvers/headerResolver.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func (hdrRes *headerResolver) RequestDataFromHash(hash []byte, _ uint32) error {
 	metaEpoch := hdrRes.epochHandler.MetaEpoch()
 	hdrRes.mutEpochHandler.RUnlock()
 
-	hdrRes.manualEpochStartNotifier.NewEpoch(metaEpoch + 1)
+	hdrRes.manualEpochStartNotifier.NewEpoch(metaEpoch + 2)
```
Comment (sasurobert) [inline dataRetriever/storageResolvers/headerResolver.go:82]: will this work always ?

**Review by sasurobert — APPROVED** [review]

File: dataRetriever/storageResolvers/headerResolver.go (hunk 1 above)
Comment (iulianpascalau) [reply dataRetriever/storageResolvers/headerResolver.go:82]: yes

Comment (iulianpascalau) [reply dataRetriever/storageResolvers/headerResolver.go:82]: it opens in advance 2 persisters rather than 1. Works on the 0->1 and x->x+1 edge cases as tested with a fetched import-db archive

**Review by sasurobert — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
System tests passed.
